### ISSUE TITLE
Eq.3 logits or probabilities

### ISSUE THREAD

**Issue by Yang-Li-2000** [tianyu0207/PEBAL#30]
In Eq.3, the PAL loss uses logits (f_theta) instead of probabilities (p_theta). When using logits, it seems that the parts inside the parenthesis can become negative. When the parts inside the parenthesis becomes negative, the log operation cannot be performed. When using probabilities, however, the parts inside the parenthesis will always be positive. Why Eq.3 uses logits instead of probabilities? 

<img width="1049" alt="image" src="https://user-images.githubusercontent.com/50163745/207316881-4e46b09c-5c1c-40a2-b1ec-32e5415186a2.png">

**Comment by yyliu01** [comment]
The equation (3) utilises the probability instead of the logits, which is close related with the equation (6) in gambler loss paper (https://arxiv.org/abs/1907.00208).

Hope it helps.
